third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -3447,6 +3571,30 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
         ExpData::Block(id, pat, binding, body)
     }
 
+    /// Digest a function entry
+    fn translate_entry_instantiation(
```

Comment:
I don't quite understand why this is needed, there is already logic to instantiate a type parameter list, its called `make_instantiation` and seems to do the same.

BTW error messages like `generic count mismatch` should not be generated manually, check `make_instantiation`.

Reply:
I missed that function.  I'll take a look.